src/fetch/payouts.py

```diff
@@ -461,7 +461,7 @@ def construct_partner_fee_payments(
 
 
 def construct_payouts(
-    dune: DuneFetcher, ignore_slippage_flag: bool, orderbook: MultiInstanceDBFetcher
+    orderbook: MultiInstanceDBFetcher, dune: DuneFetcher, ignore_slippage_flag: bool
```

Comment:
Was the reordering for aesthetic purposes?

Reply:
The flag being passed last will be for consistency as I intend config objects to be passed to some functions and they will be passed last. The slippage parameter seems to fit into that category. Or did you intend it to part of the dune fetcher?

The reordering of `orderbook` and `dune` is more due to style, as the orderbook seems the more important data source and dune will hopefully become obsolete to some degree. Since I changed the signature anyways, I changed that as well.